Wireframe/index.html

```diff
@@ -3,11 +3,40 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
+    <title>Github</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header>
+        <h1>What is Github</h1>
+        <h2>GitHub is a platform for hosting and collaborating on code projects</h2>
+      </header>
+
+      <main>
+        <section>
```

Comment:
What is this section doing? Can you explain what section "means"  in HTML?

Reply:
basically The <section> element in HTML is used to define a thematic grouping of content, and I think I didn't know what is it exactly and I tried to google it and after some research I saw and read this : https://stackoverflow.com/questions/7549561/difference-between-section-vs-article-elements-in-html
and I got conclusion that I used a main <section> that have three article in it.